doc/command_ref.rst

```diff
@@ -770,6 +770,16 @@ Remove Command
     There are also a few specific remove commands ``remove-n``, ``remove-na`` and ``remove-nevra``
     that allow to specify exact argument NEVRA format.
 
+Remove Examples
+---------------
+
+``dnf remove acpi tito``
+    Remove packages ``acpi`` and ``tito``
+
+``dnf remove $(dnf repoquery --extras --exclude=tito,acpi)``
+    Remove packages not present in any repository but it does't mark for removal packages ``tito``
```

Comment:
@ignatenkobrain There is a problem that excludes are not applied for removal, but your point is good one.